pymc/model.py

```diff
@@ -2042,16 +2042,95 @@ def Deterministic(name, var, model=None, dims=None):
 
 
 def Potential(name, var, model=None):
-    """Add an arbitrary factor potential to the model likelihood
+    """
+    Add an arbitrary factor potential to the model likelihood
+
+    The Potential function is used to add arbitrary factors (such as constraints or other likelihood components) to adjust the probability density of the model.
+
+    Warnings
+    --------
+    Potential functions only influence logp based sampling, like the one used by ``pm.sample``.
+    Potentials, modify the log-probability of the model by adding a contribution to the logp which is used by sampling algorithms which rely on the information about the observed data to generate posterior samples.
+    Potentials are not applicable in the context of forward sampling because they don't affect the prior distribution itself, only the computation of the logp.
+    Forward sampling algorithms generate sample points from the prior distribution of the model, without taking into account the likelihood function.
+    In other words, it does not use the information about the observed data.
+    Hence, Potentials do not affect forward sampling, which is used by ``sample_prior_predictive`` and ``sample_posterior_predictive``.
+    A warning saying "The effect of Potentials on other parameters is ignored during prior predictive sampling" is always emitted to alert user of this.
 
     Parameters
     ----------
-    name: str
-    var: PyTensor variables
+    name : str
+        Name of the potential variable to be registered in the model.
+    var : tensor-like
```

Comment:
Only issue in theprebiew is this isn't rendered as link: https://pymcio--6559.org.readthedocs.build/projects/docs/en/6559/api/generated/pymc.Potential.html#pymc.Potential.

Itshould usean underscore:

```suggestion
    var : tensor_like
```

Reply:
Hey @OriolAbril . Updated the docstring according to this suggestion in the latest commit.